sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -1,34 +1,194 @@
 # Azure KeyVault Administration client library for .NET
 
-Content forthcoming
+Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, or certificates. A vault is a logical group of secrets.
+
+The Azure Key Vault administration library clients supports administrative tasks such as full backup / restore and key level role based access control (RBAC).
```

Comment:
"support" - "clients" is plural and the subject of the clause, so the verb needs to conform.

Reply:
Also, "key-level, role-based access control (RBAC)". Those are compound adjectives.